### ISSUE TITLE
xpa module conflicts with imexam

### ISSUE THREAD

**Issue by olebole** [ericmandel/pyds9#58]
Original bug (Ubuntu): https://bugs.launchpad.net/bugs/1733352

Imexam does not play together with imexam. While both have overlapping use cases, it may be good that a user has the opportunity to install and use them in parallel. One way could be to make the package a subpackage of pyds9, another to agree on a common xpa package.

See also spacetelescope/imexam#122 for the imexam issue.

**Comment by montefra** [comment]
@olebole the current trunk installs also xpa stuff in a ``pyds9`` directory, so there should be no more xpa from the ``pyds9`` side laying directly in ``site-packages`` directory.

**Comment by olebole** [comment]
Any release plans?

**Comment by montefra** [comment]
I would like to release.

Most of the work that I have done has been moving ``pyds9`` to astropy utilities. I can build and test in linux and mac, but it fails to build on windows (see #44). I would like to give a try again at it and see if I can do something about it.

**Comment by olebole** [comment]
@jehturner pointed out in https://github.com/spacetelescope/imexam/issues/122#issuecomment-367823514 that the issue is solved in the current development version. Is this true? If yes, could I get the relevant commit (to include it into the Debian package), or maybe a new release?

**Comment by montefra** [comment]
yes, I think so. In the development version everything is installed under ``pyds9/`` so there should be no name collision.

If @ericmandel has nothing against it I can tag the current master as 1.9.0.rc1: would that work for you?

**Comment by olebole** [comment]
Yes. This would however mean that 1.9.0.rc1will probably (if no other release comes out before) will be part of Ubuntu 18.04 LTS.

**Comment by montefra** [comment]
when do you have the package freeze?

**Comment by ericmandel** [comment]
@montefra I have no objections to tagging the current as 1.9.0rc1.

**Comment by olebole** [comment]
@montefra [March 1](https://wiki.ubuntu.com/BionicBeaver/ReleaseSchedule) is the DebianImportFreeze.
